lessons/Early-Math/associative-property/langs/en-us/pages/which-circles-make-the-math-easier.adoc

```diff
@@ -0,0 +1,67 @@
+= Which Circle Makes the Computation Easier?
```

Comment:
I still think it's really important not to refer to circles, but to think about math concept you actually mean to refer to when you want to use the word.

In this case, I would name this page, "Which Grouping Makes the Computation Easier?"